lib/internal/errors.js

```diff
@@ -307,6 +307,7 @@ E('ERR_ENCODING_INVALID_ENCODED_DATA',
   'The encoded data was not valid for encoding %s');
 E('ERR_ENCODING_NOT_SUPPORTED', 'The "%s" encoding is not supported');
 E('ERR_FALSY_VALUE_REJECTION', 'Promise was rejected with falsy value');
+E('ERR_FS_INVALID_SYMLINK_TYPE', 'Invalid symlink type: %s');
```

Comment:
It would justify this new error a bit more if the error message includes a list of valid symlink types